python/ray/tests/test_signal.py

```diff
@@ -22,6 +27,25 @@ def receive_all_signals(sources, timeout):
             results.extend(r)
 
 
+@pytest.fixture()
```

Comment:
Test fixtures should go into `ray/python/tests/conftest.py`. In fact, you should be able to reuse one from there without adding a new one. See https://github.com/ray-project/ray/pull/4390 for more info.

Reply:
done